build/libshout-2.4.1/win32/libshout.vcxproj

```diff
@@ -96,7 +96,7 @@
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <WarningLevel>Level3</WarningLevel>
       <AdditionalIncludeDirectories>..\..\ogg\include;..\src;..\src/httpp;..\src/thread;..\src/log;..\src/avl;..\src/net;..\src/timings;../;../../pthreads;../../oggvorbis-win32sdk-1.0.1/include;../include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>_TIMESPEC_DEFINED;WIN32;NDEBUG;_LIB;_WIN32;VERSION="2.4.1";LIBSHOUT_MAJOR=2;LIBSHOUT_MINOR=0;LIBSHOUT_MICRO=0;HAVE_WINSOCK2_H;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>HAVE_OPENSSL;SHOUT_TLS=1;PTW32_STATIC_LIB;SHOUT_THREADSAFE=1;_TIMESPEC_DEFINED;WIN32;NDEBUG;_LIB;_WIN32;VERSION="2.4.1";LIBSHOUT_MAJOR=2;LIBSHOUT_MINOR=0;LIBSHOUT_MICRO=0;HAVE_WINSOCK2_H;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Sorry for the confusion -- I meant to update the SHOUT_THREADSAFE and SHOUT_TLS defines at the bottom of shout.h (they're still autoconf placeholders).